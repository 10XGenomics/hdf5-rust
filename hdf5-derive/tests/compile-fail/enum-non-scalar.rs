use hdf5_metno_derive::H5Type;

#[derive(H5Type)]
//~^ ERROR proc-macro derive
//~^^ HELP H5Type can only be derived for enums with scalar discriminants
enum Foo {
    Bar,
}

fn main() {}
