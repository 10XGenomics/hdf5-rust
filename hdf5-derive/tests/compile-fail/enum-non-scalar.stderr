error: `H5Type` can only be derived for enums with scalar discriminants
 --> tests/compile-fail/enum-non-scalar.rs:6:6
  |
6 | enum Foo {
  |      ^^^
